hack/release/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+GIT_TAG=$(git describe --exact-match --tags || echo "no tag")
```

Comment:
What if the commit has multiple tags?

Reply:
git describe "finds the most recent tag that is reachable from a commit", and I think this would actually work for annotated tags (which have dates). But let's not do that 🙂